Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```diff
@@ -368,3 +368,164 @@
     contents:
     - id: FoodDonkpocketDink
       amount: 6
+
+- type: entity
+  id: HappyHonkBase
+  parent: BoxCardboard
+  name: happy honk meal
+  description: The toy is more edible than the food.
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/Storage/happyhonk/clown.rsi
+    state: box
+  - type: Item
+    sprite: Objects/Storage/happyhonk/clown.rsi
+    heldPrefix: box
+  - type: Appearance
+    visuals:
+    - type: BagOpenCloseVisualizer
+      openIcon: box-open
+
+- type: entity
+  id: HappyHonk
+  parent: HappyHonkBase
+  suffix: Empty
+  components:
+  - type: Storage
+    capacity: 30
+    storageOpenSound:
+      collection: BikeHorn
+
+- type: entity
+  id: HappyHonkRandom
+  parent: HappyHonk
+  suffix: Filled Safe
+  components:
+  - type: StorageFill
+    contents:
+      - id: ToyMouse
+        orGroup: GiftPool
+      - id: ToyAi
+        orGroup: GiftPool
+      - id: ToyNuke
+        orGroup: GiftPool
+      - id: ToyAssistant
+        orGroup: GiftPool
+      - id: ToyGriffin
+        orGroup: GiftPool
+      - id: ToyHonk
+        orGroup: GiftPool
+      - id: ToyIan
+        orGroup: GiftPool
+      - id: ToyMarauder
+        orGroup: GiftPool
+      - id: ToyMauler
+        orGroup: GiftPool
+      - id: ToyGygax
+        orGroup: GiftPool
+      - id: ToyOdysseus
+        orGroup: GiftPool
+      - id: ToyOwlman
+        orGroup: GiftPool
+      - id: ToyDeathRipley
+        orGroup: GiftPool
+      - id: ToyPhazon
+        orGroup: GiftPool
+      - id: ToyFireRipley
+        orGroup: GiftPool
+      - id: ToyReticence
+        orGroup: GiftPool
+      - id: ToyRipley
+        orGroup: GiftPool
+      - id: ToySeraph
+        orGroup: GiftPool
+      - id: ToyDurand
+        orGroup: GiftPool
+      - id: ToySkeleton
+        orGroup: GiftPool
+      - id: FoamBlade
+        orGroup: GiftPool
+      - id: ToySkeleton
+        orGroup: GiftPool
+      - id: ClothingHeadHatBunny
+        orGroup: GiftPool
+      - id: PersonalAI
+        orGroup: GiftPool
+      - id: CrayonBox
+        orGroup: GiftPool
+      - id: ToySword
+        orGroup: GiftPool
+      - id: RevolverCapGun
+        orGroup: GiftPool
+      - id: ToyRubberDuck
+        orGroup: GiftPool
+      - id: BikeHorn
+        prob: 0.5
+        orGroup: GiftPool
+      - id: GoldenBikeHorn
+        prob: 0.1
+        orGroup: GiftPool
+
+- type: entity
+  id: HappyHonkMime
+  parent: HappyHonkRandom
+  description: A limited mime edition of the happy honk meal.
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/Storage/happyhonk/mime.rsi
+    state: box
+  - type: Item
+    sprite: Objects/Storage/happyhonk/mime.rsi
+    heldPrefix: box
```

Comment:
These don't do anything?

Reply:
the mime and cluwne meal boxes are currently just different looking boxes for the same thing, to add variety.